README.md

```diff
@@ -11,16 +11,15 @@ It provides a set of APIs to directly measure performance and behavior of your s
 
 ## Project Status
 
-| Signal  | Status     | Project               |
-|---------|------------|-----------------------|
-| Traces  | Stable     | N/A                   |
-| Metrics | Mixed [1]  | [Go: Metric SDK (GA)] |
-| Logs    | Frozen [2] | N/A                   |
+| Signal  | Status             |
+|---------|--------------------|
+| Traces  | Stable             |
+| Metrics | Stable             |
+| Logs    | In Development [1] |
 
 [Go: Metric SDK (GA)]: https://github.com/orgs/open-telemetry/projects/34
 
-- [1]: [Metrics API](https://pkg.go.dev/go.opentelemetry.io/otel/metric) is Stable. [Metrics SDK](https://pkg.go.dev/go.opentelemetry.io/otel/sdk/metric) is Beta.
-- [2]: The Logs signal development is halted for this project while we stabilize the Metrics SDK.
+- [1]: Currently the logs signal developement is in a design phase.
    No Logs Pull Requests are currently being accepted.
```

Comment:
Can I contribute to logs? :) Or PR's still not accepted?

Reply:
We would like to first have some issue or/and draft PR describing the design.

Right now, we will be focusing on stabilizing the metric exporters (at least the OTLP and console exporters). But having it done should not take too long.